File: Python Word Examples/BackgroundApiDemo/setBackgroundColor.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.background_api.BackgroundApi(configuration)

fileName = 'Rachel.docx'
color = Color(127, 255, 170)
folder = 'input'
destFilePath = 'output/SetBackgroundColor.docx'
result = api.set_background_color(name=fileName, folder=folder, color=color, dest_file_path=destFilePath)

File: JavaScript Word Examples/BackgroundApiDemo/setBackgroundColor.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.BackgroundApi(apiClient);
    
    var name = "AddBackground.docx";
    var color = new Spirecloudword.Color();
    color.red = 127;
    color.green = 255;
    color.blue = 170;
    var destFilePath = 'output/Background/SetBackgroundColor.docx';
     var opt = {folder: 'background'};
    var response = instance.setBackgroundColor(name, color, destFilePath, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();